config/prompts_memory.py

```
@@ -802,6 +802,273 @@ def get_fact_extraction_prompt(lang: str = 'zh') -> str:
 # backward compat
 fact_extraction_prompt = FACT_EXTRACTION_PROMPT['zh']
 
+# ---------- negative_preference_review_prompt → i18n dict ----------
+
+NEGATIVE_PREFERENCE_REVIEW_PROMPT = {
+    'zh': """请审查下面这段对话，判断用户是否表达了“今后不要再主动提及/推荐某个具体话题或具体事物”的稳定负面偏好。
```

Ruff (0.15.9)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)